inventories/group_vars/all/images.yaml

```diff
@@ -24,5 +24,5 @@ metal_stack_release_version: v0.11.0
 ##
 
 # ansible_common_version:
-# metal_roles_version:
+metal_roles_version: feature/auditing
```

Comment:
**Before merge:** merge ansible roles and revert this change.

Reply:
metal-roles was released.